grpc/execution/server.go

```diff
@@ -252,18 +265,54 @@ func (s *ExecutionServiceServerV1Alpha2) ExecuteBlock(ctx context.Context, req *
 				log.Debug("ignoring deposit tx with invalid asset ID", "assetID", deposit.AssetId)
 				continue
 			}
+
 			assetID := [32]byte{}
 			copy(assetID[:], deposit.AssetId[:32])
+
 			if _, ok := s.bridgeAllowedAssetIDs[assetID]; !ok {
```

Comment:
We don't check that the bridge account itself is tied to this asset or if the bridge's start height requirement has been met. Might be safer to make these checks

Reply:
good catch, added!